x/fundraising/keeper/grpc_query.go

```diff
@@ -115,26 +115,47 @@ func (k Querier) Bids(c context.Context, req *types.QueryBidsRequest) (*types.Qu
 		return nil, status.Errorf(codes.NotFound, "auction %d not found", req.AuctionId)
 	}
 
-	if req.Bidder != "" {
-		if _, err := sdk.AccAddressFromBech32(req.Bidder); err != nil {
-			return nil, err
-		}
+	var bids []types.Bid
+	var pageRes *query.PageResponse
+	var err error
+
+	store := ctx.KVStore(k.storeKey)
+	switch {
+	case req.Bidder != "" && req.Eligible == "":
+		bids, pageRes, err = queryBidsByBidder(ctx, k, store, req)
+	case req.Bidder == "" && req.Eligible != "":
+		bids, pageRes, err = queryBidsByEligible(ctx, k, store, req)
+	default:
+		bids, pageRes, err = queryAllBids(ctx, k, store, req)
```

Comment:
Wouldn't `case req.Bidder != "" && req.Eligible != ""` be possible if we add a Eligible filter option to `queryBidsByBidder`?

Reply:
Oh, I forgot the case of when a user inputs both bidder and eligible params. As you suggested, i think it is possible to add eligible filter option to the function. Let me go add the case.